File: app/src/main/java/gardezi/io/amiibofinder/binding/CustomBindingAdapter.java
package gardezi.io.amiibofinder.binding;

import android.databinding.BindingAdapter;
import android.widget.ImageView;

import com.bumptech.glide.Glide;


public class CustomBindingAdapter {

    @BindingAdapter({"bind:image"})
    public static void loadImage(ImageView imageView, String url) {
        Glide.with(imageView.getContext()).load(url).into(imageView);
    }

}

File: app/src/main/java/gardezi/io/amiibofinder/binding/CustomBindingAdapter.kt
package gardezi.io.amiibofinder.binding

import android.databinding.BindingAdapter
import android.widget.ImageView

import com.bumptech.glide.Glide

@BindingAdapter("bind:image")
fun loadImage(imageView: ImageView, url: String) {
    Glide.with(imageView.context).load(url).into(imageView)
}

